packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3973,11 +3995,49 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       : TextPosition(offset: textBoundary.getLeadingTextBoundaryAt(caretOffset) ?? 0);
   }
 
+  /// Select a paragraph around the given position.
+  ///
+  /// A paragraph boundary is defined as the range that most closely
+  /// encapsulates the given position within two line terminators. If
+  /// a line terminator does not exist in a given direction the selection
+  /// extends to the start/end of the document in that direction.
+  void selectParagraph({ required Offset position, SelectionChangedCause? cause }) {
```

Comment:
Nit: remove spaces between brackets and named parameters. Same in `selectParagraphsInRange` etc.